tests/test_coder.py

```diff
@@ -58,3 +58,8 @@ def test_coder_code_format(self, data_coder: DataCoder, first_data: DataModel):
     def test_coder_raises_if_no_code(self, data_coder: DataCoder):
         with pytest.raises(KeyError):
             data_coder.get("unknownkey")
+
+    @pytest.mark.skip("Causes infinite loop")
```

Comment:
this is bad, at the moment we cann add only 10000 records, otherwise we'll stuck forever